pkg/sqlqueries/SQL/trb_request_attendees/get_by_trb_ID.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM trb_request_attendees WHERE trb_request_id=:trb_request_id
```

Comment:
nit 
```suggestion
SELECT * FROM trb_request_attendees WHERE trb_request_id=:trb_request_id;
```

Reply:
Yeah, we sometimes do and sometimes don't which is weird. But I will add the `;`! I praise your attention to detail regardless.

Reply:
😄 thanks Lee - i may have missed it on other PRs. i'll attribute it to selective attention to detail 😉